commit 9f5a1271315b0e7c7828f2d8077ab33dca3ee8bd
Author: Florian Weimer <fweimer@redhat.com>
Date:   Thu Jul 6 14:03:39 2017 +0200

    resolv: Deal with non-deterministic address order in tst-resolv-basic

diff --git a/resolv/tst-resolv-basic.c b/resolv/tst-resolv-basic.c
index 95aea1bcfb266017..66a0e8a1659219b4 100644
--- a/resolv/tst-resolv-basic.c
+++ b/resolv/tst-resolv-basic.c
@@ -495,22 +495,6 @@ do_test (void)
   test_bug_21295 ();
   test_nodata_nxdomain ();
 
-  /* Test for bug 21295.  */
-  check_ai_hints ("www.example", "80",
-                  (struct addrinfo) { .ai_family = AF_INET6,
-                      .ai_socktype = SOCK_STREAM,
-                      .ai_flags = AI_V4MAPPED | AI_ALL, },
-                  "flags: AI_V4MAPPED AI_ALL\n"
-                  "address: STREAM/TCP 2001:db8::1 80\n"
-                  "address: STREAM/TCP ::ffff:192.0.2.17 80\n");
-  check_ai_hints ("t.www.example", "80",
-                  (struct addrinfo) { .ai_family = AF_INET6,
-                      .ai_socktype = SOCK_STREAM,
-                      .ai_flags = AI_V4MAPPED | AI_ALL, },
-                  "flags: AI_V4MAPPED AI_ALL\n"
-                  "address: STREAM/TCP 2001:db8::3 80\n"
-                  "address: STREAM/TCP ::ffff:192.0.2.19 80\n");
-
   resolv_test_end (aux);
 
   return 0;
